### PR TITLE
Improve RGBColor

### PR DESCRIPTION
**Pull request by szepeviktor**

### CODE REVIEW

File: src/RGB/RGBColor.php [hunk 1]
```diff
@@ -16,27 +16,27 @@ public function __construct(
...
 
         $highestRGBPercentage = max($percentageRed, $percentageGreen, $percentageBlue);
         $lowestRGBPercentage  = min($percentageRed, $percentageGreen, $percentageBlue);
         $chroma               = $highestRGBPercentage - $lowestRGBPercentage;
 
-        $value = 100 * $highestRGBPercentage;
-        if ((int) round($chroma) === 0) {
-            return new HSVColor(0, 0, (int) round($value));
+        $highestRGB = (int) round(100 * $highestRGBPercentage);
+        if ($chroma < 0.5) {
```
Comment (szepeviktor) [inline src/RGB/RGBColor.php:28]: 🧠 This is brain-intensive but simple!

Comment (PrinsFrank) [reply src/RGB/RGBColor.php:28]: Hi, thanks for your contribution! This line of code contained a bug resulting in all HSV colors not having a hue or saturation values. Can you rebase on master? The rest of the changes seem like a good idea!

**Review by PrinsFrank — CHANGES_REQUESTED** [review]

File: src/RGB/RGBColor.php (hunk 1 above)
Comment (szepeviktor) [reply src/RGB/RGBColor.php]: Done 🍏

**Review by PrinsFrank — APPROVED** [review]

**Comment by PrinsFrank** [comment]
Thanks again!!!

**Pull request merged**